src/main/java/org/cryptomator/frontend/webdav/mount/FallbackMounter.java

```diff
@@ -1,23 +1,23 @@
 package org.cryptomator.frontend.webdav.mount;
 
-import java.net.URI;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.net.URI;
+
 class FallbackMounter implements MounterStrategy {
 
 	private static final Logger LOG = LoggerFactory.getLogger(FallbackMounter.class);
 
 	@Override
 	public Mount mount(URI uri, MountParams mountParams) throws CommandFailedException {
-		LOG.warn("No applicable strategy has been found for your system. Please use a WebDAV client of your choice to mount: {}", uri);
-		throw new CommandFailedException("No mounting strategy found.");
+		LOG.warn("No applicable strategy has been found for your system. Please use a WebDAV client of your choice to access: {}", uri);
+		throw new CommandFailedException(new UnsupportedSystemException(uri));
```

Comment:
What purpose does this serve?

Reply:
To differentiate between the two possibilites of a failed mount operation: Either the system ist not supported or an error occurs during the mount. A downstream library can handle this case differently than a failed mount command, e.g. special error message or in combination with a gui a different screen.

Reply:
Currently I'd say it's YAGNI, since for our use cases, WebDAV is already the backup solution. But I can accept it.